website/docs/r/netapp_volume.html.markdown

```diff
@@ -0,0 +1,129 @@
+  name                 = "example-subnet"
+  resource_group_name  = "${azurerm_resource_group.example.name}"
+  virtual_network_name = "${azurerm_virtual_network.example.name}"
+  address_prefix       = "10.0.2.0/24"
+  delegation {
+    name = "testdelegation"
+  
+    service_delegation {
+      name    = "Microsoft.Netapp/volumes"
+      actions = ["Microsoft.Network/networkinterfaces/*", "Microsoft.Network/virtualNetworks/subnets/join/action"]
+    }
+  }
+}
+
+resource "azurerm_netapp_account" "example" {
+  name                = "example-netappaccount"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+}
+
+resource "azurerm_netapp_pool" "example" {
+  name                = "example-netapppool"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  account_name        = "${azurerm_netapp_account.example.name}"
+  service_level       = "Premium"
+  size_in_4_tb        = "1"
+}
+
+resource "azurerm_netapp_volume" "example" {
+  name                = "example-netappvolume"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  account_name        = "${azurerm_netapp_account.example.name}"
+  pool_name           = "${azurerm_netapp_pool.example.name}"
+  creation_token      = "my-unique-file-path"
+  service_level       = "Premium"
+  subnet_id           = "${azurerm_subnet.example.id}"
+  usage_threshold     = "100"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the NetApp Volume. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group where the NetApp Volume should be created. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `account_name` - (Required) The name of the NetApp account in which the NetApp Pool should be created.
+
+* `pool_name` - (Required) The name of the NetApp pool in which the NetApp Volume should be created.
+
+* `creation_token` - (Required) A unique file path for the volume. Used when creating mount targets.
+
+* `service_level` - (Required) The service level of the file system. Valid values include `Premium`, `Standard`, or `Ultra`.
+
+* `subnet_id` - (Required) The Azure Resource URI for a delegated subnet. Must have the delegation Microsoft.NetApp/volumes.
+
+* `usage_threshold` - (Required) Maximum storage quota allowed for a file system in bytes. This is a soft quota used for alerting only.
+
+* `export_policy_rule` - (Optional) One `export_policy_rule` block defined below.
+
+---
+
+The `export_policy_rule` block contains the following:
+
+* `rule_index` - (Required) Order index.
```

Comment:
what does this mean? could we be more descriptive here

Reply:
Specify the index number for the rule.
An export policy can consist of up to five rules. Rules are evaluated according to their order in the list of index numbers. Rules with lower index numbers are evaluated first. For example, the rule with index number 1 is evaluated before the rule with index number 2.